.circleci/config.yml

```diff
@@ -766,10 +778,12 @@ jobs:
   run-test-ios-16:
     executor:
       name: macos-executor
-      xcode_version: '15.4.0'
+      xcode_version: "15.4.0"
 
     steps:
       - checkout
+      - checkout-submodule:
+          path: upstream/paywall-preview-resources
```

Comment:
Is it necessary to checkout this submodule for `run-test-ios-16`? I'd say we only need the submodule for the `record-and-push-paywall-template-screenshots` job 🤔 (same with `run-test-ios-15`)